File: benchmark/arithmetic.py
# Integer arithmetic and tight while-loop throughput.
import time

t0 = time.perf_counter()
i = 0
value = 1

while i < 1_000_000:
    value = (value * 1_103_515_245 + i) % 1_000_000_007
    i += 1

print((time.perf_counter() - t0) * 1000)

File: benchmark/arithmetic.lua
-- Integer arithmetic and tight while-loop throughput.
local t0 = os.clock()
local i = 0
local value = 1

while i < 1000000 do
    value = (value * 1103515245 + i) % 1000000007
    i = i + 1
end

print((os.clock() - t0) * 1000)
